fedimg/services/ec2/ec2imguploader.py

```diff
@@ -166,7 +166,7 @@ def _create_volume(self, source):
         if self.volume_via_s3:
             output, err, retcode = external_run_command([
                 'euca-import-volume',
-                source,
+                source[5:],
```

Comment:
Is this to strip /tmp/ from the source name?

Reply:
yes, it is